src/main/java/frc/trigon/robot/subsystems/gripper/GripperConstants.java

```diff
@@ -246,10 +210,13 @@ private static void configureEncoder() {
 
     private static void configureLaserCAN() {
         try {
-            LASER_CAN.setRegionOfInterest(LASER_CAN_DETECTION_REGION_START_X_COORDINATE, LASER_CAN_DETECTION_REGION_START_Y_COORDINATE, LASER_CAN_DETECTION_REGION_END_X_COORDINATE, LASER_CAN_DETECTION_REGION_END_Y_COORDINATE);
-            LASER_CAN.setRangingMode(LASER_CAN_RANGING_MODE);
-            LASER_CAN.setLoopTime(LASER_CAN_LOOP_TIME);
-            LASER_CAN.setSimulationSupplier(LASER_CAN_SIMULATION_SUPPLIER);
+            LASER_CAN.setRegionOfInterest(6, 6, 12, 12);
+            LASER_CAN.setRangingMode(LaserCanInterface.RangingMode.SHORT);
+            LASER_CAN.setLoopTime(LaserCanInterface.TimingBudget.TIMING_BUDGET_33MS);
+            LASER_CAN.setSimulationSupplier(
+                    () -> SimulationFieldHandler.isCoralInGripper() &&
+                            SimulationFieldHandler.isHoldingCoral() ? 1 : 1000
+            );
```

Comment:
this should be a normal constant 
possibly the setters above, since  they don't set to variable with names